app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CollectionsController < ApplicationController
+  def index
+    if (@user = User.find_by(username: params[:username]))
+      @collections = Collection.where(user_id: @user.id)
```

Comment:
this can be: `@user.collections`

Reply:
Sounds good! And descending creation order it will be 🙂